run--01a--prep-geo-data.Rmd

```diff
@@ -260,168 +280,279 @@ schoolShp <-
-                                               2312330, 2313560, 2314100, 2314210, 2314240, 2314771, 2314793, 2314837 # list of SDs that's related to county of 005 only
-                                               # 2310770, 2311790, 2314190, 2314768, 2314789 # list of SDs that's related to county of 005 and other counties
-                                               )),
-          color = "green", 
-          fill = NA,
-          alpha = 0.2) + 
-  theme_void()
-
-#school districts that are smaller than tract
-ggplot() +
-  geom_sf(data = tractShp %>% filter(COUNTYFIP == "005"),
-          color = "red",   
-          fill = NA) +
-  geom_sf(data = schoolShp %>% filter(GEOID == 2314837),
-          color = NA, 
-          fill = "green",
-          alpha = 0.2) + 
-  theme_void()
-
-#m:m tract-SD
-ggplot() +
-  geom_sf(data = tractShp %>% filter(GEOID == "23003950100"),
-          color = "red",   
-          fill = NA) +
-  geom_sf(data = schoolShp %>% filter(school == "Caswell"),
-          color = NA, 
-          fill = "green",
-          alpha = 0.2) + 
-  geom_sf(data = schoolShp %>% filter(school == "New Sweden"),
-          color = NA, 
-          fill = "yellow",
-          alpha = 0.2) + 
-  geom_sf(data = schoolShp %>% filter(school == "School Administrative District 24"),
-          color = NA, 
-          fill = "yellowgreen",
-          alpha = 0.2) + 
-  geom_sf(data = schoolShp %>% filter(school == "Westmanland"),
-          color = NA, 
-          fill = "blue",
-          alpha = 0.2) + 
-  geom_sf(data = schoolShp %>% filter(school == "Regional School Unit 39"),
-          color = NA, 
-          fill = "purple",
-          alpha = 0.2) + 
-  geom_sf(data = schoolShp %>% filter(school == "Aroostook Unorganized Territory"),
-          color = NA, 
-          fill = "gray",
-          alpha = 0.2) + 
-  theme_void()
-
-```
-
-```{r check the 5 school districts}
-# Plot overlap to examine any potential issues for 5 municipalities/SDs:
-# "Auburn", "Bangor", "Lewiston", "Portland", "Sanford"
-# https://www2.census.gov/geo/maps/DC2020/PL20/st23_me/schooldistrict_maps/
-ggplot() +
-  geom_sf(data = schoolShp %>% filter(school == "Auburn"), 
-          color = "blue",   
-          fill = "blue") +
-  geom_sf(data = schoolShp %>% filter(school == "Bangor"), 
-          color = "green",   
-          fill = "green") +
-  geom_sf(data = schoolShp %>% filter(school == "Lewiston"), 
-          color = "yellow",   
-          fill = "yellow") +
-  geom_sf(data = schoolShp %>% filter(school == "Portland"), 
-          color = "red",   
-          fill = "red") +
-  geom_sf(data = schoolShp %>% filter(school == "Sanford"), 
-          color = "purple",   
-          fill = "purple") +
-  geom_sf(data = schoolShp,
-          color = "black",   
-          fill = NA,
-          linewidth = 0.2) +
-  theme_void()
-
-# Check with related county for each SDs
-if (FALSE) {
+```{r examine potential issues with geography for Maine}
```

Comment:
I appreciate the conditional statement here. However, I think that we should move this code to a `diagnostic` file so that it's still code that analysists can use electively, but is outside of the main code.

I wouldn't hold up this PR for us to do this, but we should make this note for ourselves for later.